### PR TITLE
review changes made

### PR DESCRIPTION
**Pull request by mohammad-premtech**

### CODE REVIEW

File: DrinksInfo.Mo/DrinksInfoAPI/DrinksInfoAPI/UI/ConsoleUI.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        Console.WriteLine("-----------------------------------");
+        if (!string.IsNullOrEmpty(drinkDetails.IdDrink))
+            Console.WriteLine($"ID: {drinkDetails.IdDrink}");
+        if (!string.IsNullOrEmpty(drinkDetails.StrDrink))
+            Console.WriteLine($"Name: {drinkDetails.StrDrink}");
+        if (!string.IsNullOrEmpty(drinkDetails.StrTags))
+            Console.WriteLine($"Tags: {drinkDetails.StrTags}");
+        if (!string.IsNullOrEmpty(drinkDetails.StrCategory))
+            Console.WriteLine($"Category: {drinkDetails.StrCategory}");
+        if (!string.IsNullOrEmpty(drinkDetails.StrIBA))
```
Comment (stevietv) [inline DrinksInfo.Mo/DrinksInfoAPI/DrinksInfoAPI/UI/ConsoleUI.cs:71]: Hi @mohammad-premtech - these two lines need a fix before the program can run. Not sure how you were able to build and test it with this in!

**Review by stevietv — CHANGES_REQUESTED** [review]

File: DrinksInfo.Mo/DrinksInfoAPI/DrinksInfoAPI/UI/ConsoleUI.cs [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            Console.WriteLine($"IBA: {drinkDetails.StrIBA}");
```
Comment (stevietv) [inline DrinksInfo.Mo/DrinksInfoAPI/DrinksInfoAPI/UI/ConsoleUI.cs:72]: Hi @mohammad-premtech - these two lines need a fix before the program can run. Not sure how you were able to build and test it with this in!

**Review by stevietv — CHANGES_REQUESTED** [review]

**Comment by stevietv** [comment]
hi @mohammad-premtech - just wanted to check in and see if you needed any help solving this?

**Comment by mohammad-premtech** [comment]
Hi @stevietv : Will make the changes today.

**Pull request closed without merging**